onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.h

```diff
@@ -20,6 +20,12 @@ struct NodeGroup {
   NodeIndex target_node;
 };
 
+struct NodeGroupNonIndex {
```

Comment:
>NodeGroupNonIndex [](http://example.com/codeflow?start=7&length=17)

are both NodeGroup and NodeGroupNonIndex required? if they can't be consolidated, consider renaming NodeGroup to NodeGroupIndices and NodeGroupNonIndex to NodeGroup